config.global.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
That file contains initial config struct, later we can extend that with some default config.